buildozer/targets/osx.py

```diff
@@ -62,9 +62,9 @@ def download_kivy(self, cwd):
                     sys.exit(1)
 
             self.logger.info('Extracting and installing Kivy...')
-            check_call(('hdiutil', 'attach', cwd + '/Kivy.dmg'))
-            buildops.file_copy(
-                '/Volumes/Kivy/Kivy.app', './Kivy.app', cwd=cwd)
+            check_call(('hdiutil', 'attach', f'{cwd}/Kivy.dmg'))
+            buildops.file_copytree(
+                '/Volumes/Kivy/Kivy.app', f'{cwd}/Kivy.app')
```

Comment:
What about using `pathlib.Path` here, instead of the f-string?

Reply:
Up to you! TBH, there like 4 different methods of joining pathes (string concatenation, `dot-foramt`, `os.path.join` (even if we work in only darwin env), fstrings). I would maybe stent some time in future just switching to `pathlib.Path` concatenation somewhere in future, not as part of this small PR (event with that I can't make it publishing for mac, but this issue  is onto osx builder package). lets make small steps?

Reply:
@butuzov: Yes, indeed. There have already been quite a few small steps towards `pathlib` in Buildozer, with more in the pipeline (eventually). So, sticking with `pathlib` here will help us not take any steps backwards. Thanks!